backend/src/app.js

```diff
@@ -14,37 +14,43 @@ import MessageTriage from './lib/message-triage'
 const app = express()
 app.disable('x-powered-by')
 app.use(function (_req, res) { res.send({ msg: 'WebSocket Only!' }) })
-app.use(morgan('combined', { stream: winston.stream }))
+app.use(morgan('combined', { stream: logger.stream }))
 
 const server = http.createServer(app)
 const socketio = io(server, { pingTimeout: 30000 })
+const genricMessage = 'message'
+const mopidyMessage = 'mopidy'
 
 MongodbService()
-
-MopidyService(socketio, mopidy => {
-  if (mopidy.playback) {
+MopidyService(
+  (data) => socketio.emit(genricMessage, data),
+  (online) => socketio.emit(mopidyMessage, Payload.toJsonString({ online })),
+  (mopidy) => {
     Scheduler.scheduleAutoPlayback({
       stop: () => mopidy.playback.stop()
     })
-  }
 
-  socketio.on('connection', socket => {
-    SocketErrorsHandler(socket)
+    socketio.on('connection', socket => {
+      socketio.emit(mopidyMessage, Payload.toJsonString({ online: true }))
```

Comment:
This is great - splitting up the messaging like this is a really nice pattern / feature of socket.io 👍